rsconf/package_data/vm_devbox/unit_override.conf.jinja

```diff
@@ -0,0 +1,4 @@
+[Service]
+RemainAfterExit=yes
```

Comment:
I think using an override for this is a problem. We are creating a new unit file, and that unit should just have this info in it instead of in two files.

Reply:
44aa09c